cmd/gh/main.go

```diff
@@ -170,15 +171,26 @@ func mainRun() exitCode {
 	rootCmd.ValidArgsFunction = func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
 		var results []string
 		if aliases, err := cfg.Aliases(); err == nil {
-			for aliasName := range aliases.All() {
+			for aliasName, aliasValue := range aliases.All() {
 				if strings.HasPrefix(aliasName, toComplete) {
-					results = append(results, aliasName)
+					s := fmt.Sprintf("%s\tAlias for %q", aliasName, aliasValue)
```

Comment:
Alias expansions, particularly for “shell“ aliases (ones that start with a `!`), can be long and span across multiple lines. With that in mind, I don't think it's a good idea to print the full alias expansion. What to you think about the following:

- For shell aliases we don't print the expansion text: we just print “Shell alias” as description;
- For other aliases, we truncate the expansion to a set max length to avoid having long descriptions.